nebula/ngMeta/MetaClient.py

```diff
@@ -0,0 +1,283 @@
+
+        return None
+
+    def getTags(self, spaceName):
+        spaceId = self.getSpaceIdFromCache(spaceName)
+        if spaceId == -1:
+            return None
+        listTagsReq = ListTagsReq(spaceId)
+        listTagsResp = self.metaClient.listTags(listTagsReq)
+        
+        if listTagsResp.code == ErrorCode.SUCCEEDED:
+            return listTagsResp.tags
+        else:
+            print('get tags error, error code: ', listTagsResp.code)
+            return None
+
+    def getTag(self, spaceName, tagName):
+        spaceId = self.getSpaceIdFromCache(spaceName)
+        getTagReq = GetTagReq(spaceId, tagName, -1)
+        getTagResp = self.metaClient.getTag(getTagReq)
+
+        if getTagResp.code == ErrorCode.SUCCEEDED:
+            return getTagResp.schema
+        else:
+            return None
+
+    def getTagSchema(self, spaceName, tagName, version=-1):
+        spaceId = self.getSpaceIdFromCache(spaceName)
+        getTagReq = GetTagReq(spaceId, tagName, version)
+        getTagResp = self.metaClient.getTag(getTagReq)
+        result = {}
+        for columnDef in getTagResp.schema.columns:
+            result[columnDef.name] = columnDef.type.type # 需要将type convert to Class
+
+        return result
+
+    def getEdgeItemFromCache(self, spaceName, edgeName):
+        if spaceName not in self.spaceEdgeItems.keys():
+            edges = {}
+            for edgeItem in self.getEdges(spaceName):
+                edges[edgeItem.edge_name] = edgeItem
+            self.spaceEdgeItems[spaceName] = edges
+
+        edgeItems = self.spaceEdgeItems[spaceName]
+        if edgeName in edgeItems.keys():
+            return edgeItems[edgeName]
+        else:
+            return None
+    
+    def getEdgeNameFromCache(self, spaceName, edgeType):
+        if spaceName in self.edgeNameMap.keys():
+            edgeNames = self.edgeNameMap[spaceName]
+            if edgeType in edgeNames.keys():
+                return edgeNames[edgeType]
+
+        return None
+
+    def getEdges(self, spaceName):
+        spaceId = self.getSpaceIdFromCache(spaceName)
+        if spaceId == -1:
+            return None
+        listEdgesReq = ListEdgesReq(spaceId)
+        listEdgesResp =self.metaClient.listEdges(listEdgesReq)
+        if listEdgesResp.code == ErrorCode.SUCCEEDED:
+            return listEdgesResp.edges
+        else:
+            print('get tags error, error code: ', listEdgesResp.code)
+            return None
+
+    def getEdge(self, spaceName, edgeName):
+        spaceId = self.getSpaceIdFromCache(spaceName)
+        if spaceId == -1:
+            return None
+        getEdgeReq = GetEdgeReq(spaceId, edgeName, -1)
+        getEdgeResp = self.metaClient.getEdge(getEdgeReq)
+        if getEdgeResp.code == ErrorCode.SUCCEEDED:
+            return getEdgeResp.Schema
+        else:
+            print('get edge error, error code: ', getEdgeResp.code)
+            return None
+
+    def getEdgeSchema(self, spaceName, edgeName, version=-1):
```

Comment:
`getEdgeSchema` can call `getEdge` directly.
why need two interfaces?

Reply:
These 2 interfaces return a different result, I think these 2 interfaces are both useful for users